CC/Sounder/recorder.cc

```diff
@@ -26,7 +26,13 @@ const int DS_SIM = 5;
 Recorder::Recorder(Config* cfg)
 {
     this->cfg = cfg;
-    size_t buffer_chunk_size = SAMPLE_BUFFER_FRAME_NUM * cfg->symbolsPerFrame * cfg->getNumAntennas();
+    //size_t buffer_chunk_size = SAMPLE_BUFFER_FRAME_NUM * cfg->symbolsPerFrame * cfg->getNumAntennas();
+    size_t buffer_chunk_size;
```

Comment:
We can make this a class variable since it is used in the `record` function too.

Reply:
done.
and reformatted the code as well